src/components/interior-left-nav/_interior-left-nav.scss

```diff
@@ -8,6 +8,37 @@
 @import 'carbon-components/scss/globals/scss/css--reset';
 @import 'carbon-components/scss/globals/scss/typography';
 
+body.bx--v9-theme {
+  @include exports('cloud-interior-left-nav') {
```

Comment:
- [ ] One last thing and this should be good to go. I would remove this mixin and add them to the scoped styles to the end of the styles contained within the mixin on [line 42](https://github.com/carbon-design-system/carbon-addons-cloud/pull/126/files#diff-8b6b14205d45f8092726a1f310743845R42). 

Leaving it like this will only export these styles and miss out on all of the other styles, I believe.

Reply:
I've done that, just forgot to commit 🙄 Pushed it, once the check completes I'll merge it. Thank you very much.

Reply:
Ah, nope, I don't have the permissions to merge it 😄